Interfaces/API/Job.py

```diff
@@ -578,39 +579,33 @@ def setDestination( self, destination ):
     """
     kwargs = {'destination':destination}
     if isinstance( destination, basestring ):
-      if not re.search( '^DIRAC.', destination ) and not destination.lower() == 'any':
-        result = self.__checkSiteIsValid( destination )
+      destination = [destination]
+      description = 'User specified destination site'
+    else:
+      description = 'List of sites selected by user'
```

Comment:
This is the way it was ... I am not sure this "description" is used anywhere in fact, but just to be safe I left it...